code/__DEFINES/dcs/flags.dm

```diff
@@ -7,6 +7,8 @@
 
 /// Return value to cancel attaching
 #define ELEMENT_INCOMPATIBLE 1
+/// Returned in PostTransfer to prevent transfer, similar to `COMPONENT_NOTRANSFER`
+#define ELEMENT_NOTRANSFER 2
```

Comment:
Elements don't transfer?

Reply:
Elements do not transfer. In fact, datums which they're attached to should hold no reference to element outside of signals.